master/buildbot/www/rest.py

```diff
@@ -61,7 +61,7 @@ def __init__(self, contenttype):
         self.typeheader = contenttype
 
     def gettype(self):
-        mimetype, _ = cgi.parse_header(bytes2unicode(self.typeheader))
+        mimetype, _ = parse_header(bytes2unicode(self.typeheader))
```

Comment:
Looks like this could simply be replaced with
```python
if self.typeheader is None:
    return None
return bytes2unicode(self.typeheader).split(';', 1)[0]
```

[ref](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type#syntax)